### DISCUSSION TITLE
Using with Remix run

### DISCUSSION THREAD

**Discussion by CibiAananth** [rsuite/rsuite#3082] [Q&A] [answered]
How to use rsuite with Remix run. Is there a document with steps to integrate with Remix?

**Comment by simonguo** [comment, marked as answer]
There is currently no guide on Remix. But I think rsuite can be well integrated in Remix as a UI component, and we will consider adding an example later.

**Reply by CibiAananth** [reply]
Thanks @simonguo . Looking forward.

**Reply by simonguo** [reply]
https://github.com/rsuite/rsuite/tree/main/examples/with-remix
